torch/_higher_order_ops/associative_scan.py

```diff
@@ -77,11 +79,14 @@ def add(x: torch.Tensor, y: torch.Tensor):
     if not torch._dynamo.is_compiling():
         with _set_compilation_env(), torch._dynamo.utils.disable_cache_limit():
             return torch.compile(associative_scan, fullgraph=True)(
-                combine_fn, input, dim
+                combine_fn, input, dim, reverse=reverse
             )
 
     leaves, spec = pytree.tree_flatten(input)
 
+    if reverse:
+        leaves = [torch.flip(elem, [dim]) for elem in leaves]
```

Comment:
In practice `elem.flip(dim)` also works but unfortunately this is not documented properly:
- https://github.com/pytorch/pytorch/issues/68616